assets/src/scripts/monitoring-location/components/hydrograph/parameters.js

```diff
@@ -154,7 +187,7 @@ const drawRadioButtonRow = function(container, parameter, store) {
     const gridRowInnerWithRadioButton = container.append('div')
         .attr('class', 'grid-row grid-row-inner');
```

Comment:
Can we try to use mixins rather than utility classes for grid-row?

Reply:
Darn, forgot to change those. I have to be honest the mixins are neat, but I find it hard to set up the initial structure of grids and similarly complex CSS  using them from the start because it makes it harder for me to see what grid row is holding which columns. So I tend to set things up with the utility classes and than convert them over. And I always seem to miss a few.

Reply:
Well, I thought about this over the night, and I'm not sure if using the @include for the 'grid-rows' would make things more clear. Since the 'grid-row' is the only class on these divs (yes you were right, the 'grid-row-inner is not used), I don't have a good way to identify them in the CSS so that the @include can be used. If I kept the 'grid-row-inner', I could use that as the selector and attach the @include to that, but that seems like I'm only trading one class for another.